src/platforms/shopware/composables/useCustomer.ts

```diff
@@ -92,6 +93,7 @@ export const useCustomer = defineStore('use-customer', (): HblIUseCustomer => {
             const platformStore = usePlatform()
             const { session } = storeToRefs(platformStore)
             const { getCart } = useCart()
+            const { getWishlist } = useWishlist()
```

Comment:
useWishlist composable is used multiple times in this composable, so you can init the wishlist composable once at the beginning of this composable outside of the functions and reuse it in the functions where needed. Same we should try with useCart composable